perfkitbenchmarker/publisher.py

```diff
@@ -185,6 +185,9 @@ def AddMetadata(self, metadata, benchmark_spec):
           for key, value in data_disk.metadata.iteritems():
             metadata[name_prefix + 'data_disk_0_' + key] = value
 
+    metadata['set_files'] = ','.join(FLAGS.set_files)
```

Comment:
I think it might be nicer not to add a metadata key unless we actually modified something (set a file or sysctl) since in the vast majority of cases things wont be modified